lfc/core/src/main/kotlin/org/lflang/generator/uc/UcFederatedProjectTemplateGenerator.kt

```diff
@@ -72,21 +72,28 @@ class UcFederatedTemplateGenerator(private val mainDef: Instantiation, private v
         val make = """
             |LF_MAIN ?= ${mainDef.name}
             |LF_FED ?= ${federate.name}
-            |LF_SRC_PKG_DIR ?= ${projectRoot}/../
             |
             |# Execute the LF compiler if build target is "all"
-            |ifeq ($S{'$'}(firstword $S{'$'}(MAKECMDGOALS)),all)
-            | _ :=  $S{'$'}(shell $S{'$'}(REACTOR_UC_PATH)/lfc/bin/lfc-dev $S{LF_SRC_PKG_DIR}/src/$S{'$'}(LF_MAIN).lf -n -o ${projectRoot})
+            |ifeq ($S(firstword $S(MAKECMDGOALS)),all)
+            | _ :=  $S(shell $S(REACTOR_UC_PATH)/lfc/bin/lfc-dev $S(CURDIR)/../../src/$S(LF_MAIN).lf -n -o $S(CURDIR))
```

Comment:
I made these changes because I think having relative paths is better than having a hardcoded `/var/home/lasse` path.

This would make it very difficult to collaborate on a lingua franca projekt